sentry-kotlin-multiplatform/src/androidUnitTest/kotlin/io/sentry/kotlin/multiplatform/BaseSentryTest.kt

```diff
@@ -0,0 +1,17 @@
+package io.sentry.kotlin.multiplatform
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(sdk = [28])
```

Comment:
Is there a reason to use 28 specifically?

Reply:
It really depends if you are using specific features that depend on OS specific, otherwise, it's not important.